doc/sphinx/inter.rst

```diff
@@ -233,41 +233,46 @@ on/off during a simulation.
 Lennard-Jones cosine interaction
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-.. todo::
-    
-    Not implemented yet.
+.. note::
 
+   `Feature LJCOS and/or LJCOS2 required.`
 
-inter lj-cos inter lj-cos2
+.. code::
 
-specifies a Lennard-Jones interaction with cosine
-tail :cite:`soddeman01a` between particles of the types *type1* and *type2*
-. The first variant behaves as follows: Until the minimum of the
-Lennard-Jones potential at
-:math:`r_\mathrm{min} = r_\mathrm{off} +
-2^{\frac{1}{6}}\sigma`, it behaves identical to the unshifted
-Lennard-Jones potential (:math:`c_\mathrm{shift}=0`). Between and
-, a cosine is used to smoothly connect the potential to 0,
+   system.non_bonded_inter[type1, type2].lennard_jones_cos.set_params(**kwargs)
+   system.non_bonded_inter[type1, type2].lennard_jones_cos2.set_params(**kwargs)
 
-.. math:: V(r)=\frac{1}{2}\epsilon\left(cos\left[\alpha(r - r_\mathrm{off})^2 + \beta\right]-1\right),
+:class:`espressomd.interactions.LennardJonesCosInteraction` and
+:class:`espressomd.interactions.LennardJonesCos2Interaction` specifies
+a Lennard-Jones interaction with cosine tail :cite:`soddeman01a`
+between particles of the types *type1* and *type2* . The first variant
+behaves as follows: Until the minimum of the Lennard-Jones potential
+at :math:`r_\mathrm{min} = r_\mathrm{off} + 2^{\frac{1}{6}}\sigma`, it
+behaves identical to the unshifted Lennard-Jones potential
+(:math:`c_\mathrm{shift}=0`). Between and , a cosine is used to
```

Comment:
Between `` :math:`r_\mathrm{min}` `` and `` :math:`r_\mathrm{cut}` ``, ...
(see [ESPResSo User’s Guide](http://espressomd.org/wordpress/wp-content/uploads/2017/06/ESPResSo_3.3.1_UsersGuide.pdf), Section 5.1.4. Lennard-Jones cosine interaction)